### PR TITLE
Feature/enablemodule

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [x] I have read and agreed to the [CODE OF CONDUCT](https://github.com/CascadeBot/CascadeBot/blob/master/CODE_OF_CONDUCT.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [trello](https://trello.com/b/gUQehy4l/bot).
 - [x] I have tested all of my changes.
 - [x] I have followed the style of the rest of the project.
 
## Added/Changed
What did you add and/or change for this.

## Feature
If this is a new feature, why should we add it to the bot.

### CODE REVIEW

File: src/main/java/com/cascadebot/cascadebot/commands/core/ModuleCommand.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import com.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import net.dv8tion.jda.core.entities.Member;
+
+import java.util.Set;
+
+public class ModuleCommand implements ICommandCore {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+
```
Comment (weeryan17) [inline src/main/java/com/cascadebot/cascadebot/commands/core/ModuleCommand.java]: Add some sort of usage here so when people run the command they have something.

**Comment by weeryan17** [comment]
Also add permissions to everything.

**Comment by JamesHawkinss** [comment]
> Also add permissions to everything.

This is a core command xoxo

**Comment by weeryan17** [comment]
> 
> 
> > Also add permissions to everything.
> 
> This is a core command xoxo

It also is a command we don't want everyone to be able to run

**Comment by JamesHawkinss** [comment]
> > > Also add permissions to everything.
> > 
> > 
> > This is a core command xoxo
> 
> It also is a command we don't want everyone to be able to run

@binaryoverload has dealt with this, permissions have been created and so forth xoxo

**Comment by weeryan17** [comment]
> 
> 
> > > > Also add permissions to everything.
> > > 
> > > 
> > > This is a core command xoxo
> > 
> > 
> > It also is a command we don't want everyone to be able to run
> 
> @binaryoverload has dealt with this, permissions have been created and so forth xoxo

Actually i created them first binary just fixed them lol

**Comment by weeryan17** [comment]
@binaryoverload we need to get a proper usage system up and running. I can handle it this time if you want.

**Comment by binaryoverload** [comment]
@weeryan17 I'm good for you to do usage, maybe a better way to construct instead of jsut strings? Happy to see what you can come up with

**Pull request merged**